### PR TITLE
Enhanced ffmpeg WEBM converter to handle even file resolution. Reduced output file size

### PR DESCRIPTION
**Pull request by awkdev**

### CODE REVIEW

File: src/handlers/webm_converter.py [hunk 1]
```diff
@@ -22,23 +23,32 @@ def handle(bot, update):
...
+        minutes = int(difference.seconds/60)
+        seconds = difference.seconds%60
         og_from = update.message.from_user
 
         if(og_from['username']):
             og_sender = "@" + og_from['username']
         elif(og_from['firstname']):
             og_sender = og_from['firstname']
 
-        caption = "madarchod " + og_sender + " kal se webm cancel!"
```
Comment (vcidst) [inline src/handlers/webm_converter.py]: I object to this change

Comment (awkdev) [reply src/handlers/webm_converter.py]: `smd` bro

Comment (aman-conceptowl) [reply src/handlers/webm_converter.py]: Don't merge

**Comment by meamank** [comment]
Don't do it

**Pull request merged**